sickrage/providers/GenericProvider.py

```diff
@@ -523,14 +523,19 @@ def _verify_download(self, file_name=None):  # pylint: disable=unused-argument,n
     @property
     def recent_results(self):
         if sickbeard.provider_recent_results.get(self.get_id()):
-            return sickbeard.provider_recent_results.get(self.get_id())[::-1]
+            return sickbeard.provider_recent_results.get(self.get_id())
```

Comment:
With a `get()` you don't need an if statement, just pass the default value, `[]` in this case, as the second arg to `get()`